gui_views/state.py

```diff
@@ -39,3 +39,15 @@ def drawConnection(self, p, connection):
         if connection.line:
             self.scene.removeItem(connection.line)
         connection.line = line
+
+
+def get_path():
+    cur_dir = os.getcwd()
+    os.chdir("/home") #make just / if not assuming gem5 in home directory
+
+    for root, dirs, files in os.walk(os.getcwd(), topdown=False):
```

Comment:
This could take a very long time on systems with lots of files. For instance, amarillo's /home is a 10TB RAID array.

I wouldn't try to discover manually like this. Maybe take where the executable (gem5.opt) is and checking one directory up?

Reply:
ah ok i'll revise it!